timesketch/frontend/src/views/Attributes.vue

```diff
@@ -0,0 +1,82 @@
+<!--
+Copyright 2021 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <div>
+    <ts-navbar-main>
+      <template v-slot:left>
+        {{ sketch.name }}
+      </template>
+    </ts-navbar-main>
+
+    <ts-navbar-secondary
+      currentAppContext="sketch"
+      currentPage="attributes"
+    ></ts-navbar-secondary>
+
+    <section v-if="meta.permissions.read" class="section">
```

Comment:
You will not be able to get this far if you don't have read access to the sketch, so this check should be safe to remove.